tools/flang1/flang1exe/module.c

```diff
@@ -372,6 +373,11 @@ apply_use_stmts(void)
 
   if (!gbl.currmod && gbl.internal <= 1) {
     init_use_tree();
+    for (m_id = FIRST_USER_MODULE; m_id < usedb.avl; m_id++) {
+      if(usedb.base[m_id].scope > sem.scope_level){
+        remove_from_use_tree(SYMNAME(usedb.base[m_id].module));
```

Comment:
Since `init_use_tree` set `use_tree` to be NULL, `remove_from_use_tree` seems to cause no side effects.

Reply:
you are right- its fixed- i removed init_use_tree